integration/default_test.go

```diff
@@ -68,16 +74,21 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 			)
 
 			image, logs, err = pack.WithNoColor().Build.
-				WithPullPolicy("never").
-				WithBuildpacks(buildpack, buildPlanBuildpack).
+				WithExtensions(
+					settings.Extensions.UbiNodejsExtension.Online,
+				).
+				WithBuildpacks(
+					settings.Buildpacks.Yarn.Online,
+					settings.Buildpacks.BuildPlan.Online,
+				).
```

Comment:
Asked @pacostas about this change, it was done to be consistent with all of the other buildpack integration tests.